test/common/EnvVars.cpp

```diff
@@ -63,6 +63,7 @@ namespace RcclUnitTesting
     maxRanksPerGpu = GetEnvVar("UT_MAX_RANKS_PER_GPU", 1);
     showTiming     = GetEnvVar("UT_SHOW_TIMING",  1);
     useInteractive = GetEnvVar("UT_INTERACTIVE",  0);
+    timeout        = GetEnvVar("UT_TIMEOUT"    ,  0);
```

Comment:
Set this to 5 seconds.  No collective should take longer than 5 seconds.

Reply:
Acceptable time out is about 100X of normal latency which is max 50us with 5KB data size. Test can eventually pass at 5 seconds. Thus 5ms is proper timeout for protocol issues identification.